.qube.yml

```diff
@@ -3,7 +3,7 @@ full_name: Jennifer Bödker
 email: jennifer.boedker@qbic.uni-tuebingen.de
 project_name: Sample Tracking Status Overview
 project_short_description: Gives a visual overview of sample statuses
-version: 0.2.1
+version: 0.3.0-SNAPSHOT
```

Comment:
@jenniferboedker why is this in here and not on development o.O

Reply:
I guess that was a mistake from my side

Reply:
Feel free to remove that here and add it to dev :)